### PR TITLE
rename package to `supabase` from `netlify`

### PR DESCRIPTION
**Pull request by hf**
After forking the repo from https://github.com/netlify/gotrue, we need to change the Go package name to point to this repository.

### CODE REVIEW

File: Dockerfile.postgres.dev [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM postgres:14
+FROM postgres:15
```
Comment (kangmingtay) [inline Dockerfile.postgres.dev:1]: maybe do this separately next time, could be easy to miss amidst the bunch of other changes going in

**Review by kangmingtay — APPROVED** [review]

File: Dockerfile.postgres.dev (hunk 1 above)
Comment (J0) [reply Dockerfile.postgres.dev:1]: Good call - cherry-picked to [another branch](https://github.com/supabase/gotrue/pull/882) that can go in first and should be good after

File: Dockerfile [hunk 2]
```diff
@@ -5,24 +5,24 @@ ENV GOOS=linux
...
 # Pulling dependencies
 COPY ./Makefile ./go.* ./
 RUN make deps
 
 # Building stuff
-COPY . /go/src/github.com/netlify/gotrue
+COPY . /go/src/github.com/supabase/gotrue
 RUN make build
 
 FROM alpine:3.15
```
Comment (J0) [inline Dockerfile:18]: Should probably bump this too - opened a [PR](https://github.com/supabase/gotrue/pull/883)

File: docker-compose-dev.yml [hunk 3]
```diff
@@ -11,10 +11,10 @@ services:
       - '9999:9999'
       - '9100:9100'
```
Comment (J0) [inline docker-compose-dev.yml:12]: Just noticing this - does anyone know what the 9100 port is used for?

**Review by J0 — APPROVED** [review]
lgtm, thanks for putting this together @hf ! Think we'll eventually have to rename again to `github.com/supabase/<new_name>/<package>` if we decide to rename to something other than `gotrue` but I think that can be left for another time.

**Pull request closed without merging**